### PR TITLE
Switch lobby music

### PR DESCRIPTION
**Pull request by dolgovmi**
## About the PR
Changed lobby music to Story 1 by Project Moon.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/Audio/Lobby/attributions.yml [hunk 1]
```diff
@@ -68,3 +68,9 @@
   license: "CC-BY-NC-SA-3.0"
   copyright: "every light is blinking at once by Alexander Divine."
   source: "https://soundcloud.com/alexanderdivine/every-light-is-blinking-at-once"
+
+ files: ["Story1.ogg"]
```
Comment (PixelTheKermit) [inline Resources/Audio/Lobby/attributions.yml:72]: You need a - in front.

**Comment by dolgovmi** [comment]
why test fail 😢

**Comment by dolgovmi** [comment]
why the fuck is it failing though

**Comment by dolgovmi** [comment]
wait until given or denied permission to use

**Pull request closed without merging**